examples/DS2434_IBM701c/DS2434_IBM701c.ino

```diff
@@ -28,8 +28,7 @@ void setup()
     ds2434.writeMemory(reinterpret_cast<const uint8_t *>(mem2),sizeof(mem2),0x20);
 
     ds2434.lockNV1();
-    ds2434.setID(0xCABDu);
```

Comment:
the examples usually try to show usage of every function of the class. but I can also understand, that storage is tight

Reply:
Good call. I think leaving it in and showing all options is the right way to go. I can create one tuned for an attiny implementation in my own fork.